roles/download/defaults/main.yml

```diff
@@ -1218,6 +1218,51 @@ downloads:
     groups:
     - k8s_cluster
 
+  cilium_hubble_relay:
+    enabled: "{{ cilium_enable_hubble and cilium_hubble_install }}"
```

Comment:
Necessary to define the default value of `cilium_enable_hubble` to avoid the following error at the CI:
```
'cilium_enable_hubble' is undefined
he error appears to be in '/builds/kargo-ci/kubernetes-sigs-kubespray/roles/download/tasks/download_file.yml': line 7, column 5, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:\n
  - name: download_file | Starting download of file
    ^ here
```

Reply:
@oomichi 😁 thanks for the reminder, I have added the definition of the default value.